types/jsforce/api/chatter.d.ts

```diff
@@ -43,6 +43,8 @@ export class Request<T> implements Promise<T> {
 
     thenCall(callback?: (err: Error, records: T) => void): Query<T>;
 
+    finally(onfinally?: (() => void) | null | undefined): Promise<T>;
```

Comment:
Why the null and undefined here since you already have the optional modifier?

Reply:
I'm just using the same signature as the official implementation, as you can see here https://github.com/Microsoft/TypeScript/blob/master/lib/lib.es2018.promise.d.ts#L31